### ISSUE TITLE
Font setting error (on init) for Tabs and Debug Console after f035d4f3

### ISSUE THREAD

**Issue by bitraid** [qutebrowser/qutebrowser#3096]
Let's assume a simple config.py:
```
c.fonts.monospace = '"xos4 Terminus"'
c.fonts.tabs = '14px monospace'
c.fonts.debug_console = '14px monospace'
```
I get this error:
```
Errors occurred while reading config.py: 
Unhandled exception: 'NoneType' object has no attribute 'replace' 
Traceback (most recent call last):
  File "/home/bitraid/bin/qutebrowser/qutebrowser/config/configfiles.py", line 406, in read_config_py
    exec(code, module.__dict__)
  File "/home/bitraid/.config/qutebrowser/config.py", line 3, in <module>
    c.fonts.debug_console = '14px monospace'
  File "/home/bitraid/bin/qutebrowser/qutebrowser/config/config.py", line 460, in __setattr__
    self._config.set_obj(name, value)
  File "/home/bitraid/bin/qutebrowser/qutebrowser/config/config.py", line 323, in set_obj
    self._set_value(self.get_opt(name), value)
  File "/home/bitraid/bin/qutebrowser/qutebrowser/config/config.py", line 257, in _set_value
    opt.typ.to_py(value)  # for validation
  File "/home/bitraid/bin/qutebrowser/qutebrowser/config/configtypes.py", line 1077, in to_py
    family = family.replace('"', '').replace("'", '')
AttributeError: 'NoneType' object has no attribute 'replace'
```
If i do a config-source after qutebrowser starts the values are read ok.

**Comment by The-Compiler** [comment]
Whoops... Looks like a v1.0.1 will happen rather sooner than later.

**Comment by The-Compiler** [comment]
Also affects autoconfig.yml:

```
Traceback (most recent call last):
  File "/usr/bin/qutebrowser", line 11, in <module>
    load_entry_point('qutebrowser==1.0.0', 'gui_scripts', 'qutebrowser')()
  File "/usr/lib/python3.6/site-packages/qutebrowser/qutebrowser.py", line 186, in main
    return app.run(args)
  File "/usr/lib/python3.6/site-packages/qutebrowser/app.py", line 100, in run
    configinit.early_init(args)
  File "/usr/lib/python3.6/site-packages/qutebrowser/config/configinit.py", line 61, in early_init
    configfiles.read_autoconfig()
  File "/usr/lib/python3.6/site-packages/qutebrowser/config/configfiles.py", line 423, in read_autoconfig
    config.instance.read_yaml()
  File "/usr/lib/python3.6/site-packages/qutebrowser/config/config.py", line 268, in read_yaml
    self._set_value(self.get_opt(name), value)
  File "/usr/lib/python3.6/site-packages/qutebrowser/config/config.py", line 257, in _set_value
    opt.typ.to_py(value)  # for validation
  File "/usr/lib/python3.6/site-packages/qutebrowser/config/configtypes.py", line 1077, in to_py
    family = family.replace('"', '').replace("'", '')
AttributeError: 'NoneType' object has no attribute 'replace'
```

**Comment by The-Compiler** [comment]
As a workaround, do this in a `~/.config/qutebrowser/config.py`:

```python
from qutebrowser.config import configtypes
configtypes.Font.monospace_fonts = c.fonts.monospace
config.load_autoconfig()
```

**Comment by The-Compiler** [comment]
Fixed in `master` and the `v1.0.x` branch now. Leaving this open for visibility for now, and I'm probably going to release a v1.0.1 later today.

**Comment by The-Compiler** [comment]
v1.0.1 is uploading.
